package.json

```diff
@@ -31,7 +31,8 @@
     "test": "npm run test-only && npm run check-types && npm run eslint && npm run prettier",
     "test-only": "cross-env JEST_IMAGE_SNAPSHOT_TRACK_OBSOLETE=1 vitest run --coverage",
     "tsc": "npm run clean && npm run tsc-build",
-    "tsc-build": "tsc --project tsconfig.build.json"
+    "tsc-build": "tsc --project tsconfig.build.json",
+    "postinstall": "allow-scripts run"
```

Comment:
Is `postinstall` the recommended place for it?
It doesn't make sense to me because that would be run by end-users where it doesn't make sense.

Reply:
No, it's not a recommended place, I was trying to find an option for the `lavamoat` to run with `npm install` but I don't know how it can be done. Adding its config and workspace to package.json did nothing. Am I misunderstanding something?